implementation/UserManipulation.py

```
@@ -11,11 +13,10 @@ def insert(self,userData):
                     userData['password'],
                     userData['confirm_password'],
-                    userData['mail']
-            )
-            self.cursor.collection('user_model_table').document(userData['mail']).set(user_data.__str__()) 
-        def update(self,userUpdateData,email:str):
-            
-            self.cursor.collection('user_model_table').document(email).update(userUpdateData)
+                    userData['mail'])
+            self.user_model=user_data.__str__()
+           
+            return requests.post("http://localhost:7006/insert",json=user_data.__str__()).json()
```

Flake8 (7.2.0)

[error] missing whitespace after ','

(E231)